classes/PrestaShopAutoload.php

```diff
@@ -154,7 +154,10 @@ public function generateIndex()
 		if ($filename_tmp !== false && file_put_contents($filename_tmp, $content) !== false)
 		{
 			if (!@rename($filename_tmp, $filename))
+			{
```

Comment:
Hi tchauviere,

this is solved
I have not changed the tabs in the space , to not break the indentation file.